quilt/tools/build.py

```diff
@@ -135,38 +167,59 @@ def _callback(count):
         failover_args = {}
         failover_args.update(failover)
         failover_args.update(kwargs)
-        df = handler(path, **failover_args)
+        dataframe = handler(path, **failover_args)
 
     # cast object columns to strings
-    for name, col in df.iteritems():
+    for name, col in dataframe.iteritems():
         if col.dtype == 'object':
-            df[name] = col.astype(str)
+            dataframe[name] = col.astype(str)
 
-    return df
+    return dataframe
 
-def build_package(username, package, yaml_path):
+def build_package(username, package, yaml_path, checks_path=None, dry_run=False, env='default'):
     """
     Builds a package from a given Yaml file and installs it locally.
 
     Returns the name of the package.
     """
+    def find(key, value):
+        """find all nodes transitively"""
+        for k, v in value.items():
```

Comment:
Use `iteritems` from `six`

Reply:
DONE re list()...
NOCHG re iteritems - I need transitive iteration, iteritems() is shallow.

Reply:
What do you mean by transitive iteration? `six.iteritems` should be the same as `iteritems` in Python2 and `items` in Python3.

Reply:
DONE: haha I thought you meant all of find() not just this one line.